lambdas/handlers/logout_handler.py

```diff
@@ -40,15 +41,23 @@ def logout_handler(token):
 
     except ClientError as e:
         logger.error(f"Error logging out user: {e}", {"Result": "Unsuccessful logout"})
+        error = LambdaError.LogoutClient.value
+        msg = error["message"]
+        err_code = error["err_code"]
         return ApiGatewayResponse(
-            500, "Error logging user out", "GET"
+            500,
+            ErrorResponse(err_code, msg).create(),
+            "GET",
         ).create_api_gateway_response()
     except (jwt.PyJWTError, KeyError) as e:
         logger.error(
             f"error while decoding JWT: {e}", {"Result": "Unsuccessful logout"}
         )
+        error = LambdaError.LogoutAuth.value
```

Comment:
Since the error var is created just to pass the `msg` and `err_code` to` ErrorResponse`, is there anyway we can just pass it an error class and the `ErrorResponse` can then extract the `msg` and `err_code` in it's create() method? Would reduce a lot of duplication from what I can see, something like `ErrorResponse(LambdaError.LogoutAuth).create()`